debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -314,6 +327,71 @@ public boolean exists() {
         return exists;
     }
 
+    private void checkTopicSettings(String topicName) {
+        final Thread t = new Thread((Runnable) () -> {
+            try (AdminClient admin = AdminClient.create(this.producerConfig.asProperties())) {
+
+                Set<ConfigResource> resources = Collections.singleton(new ConfigResource(ConfigResource.Type.TOPIC, topicName));
+                final Map<ConfigResource, Config> configs = admin.describeConfigs(resources).all().get(
+                        KAFKA_QUERY_TIMEOUT.toMillis(), TimeUnit.MILLISECONDS);
+                if (configs.size() != 1) {
+                    logger.info("Expected one topic '{}' to match the query but got {}", topicName, configs.values().size());
```

Comment:
Can you change it to static `LOGGER` while at it?

Reply:
Done